library/src/skins/default/src/sass/modules/more-sites/_more-sites.scss

```
@@ -21,6 +17,34 @@ body.active-more-sites {
     padding: 5px;
   }
 
+  .card-title {
+    .bi-pin-fill {
+      color: var(--sakai-text-color-1);
+    }
+
+    &:hover, &:focus, button:focus {
+      .bi-pin {
+        color: var(--sakai-text-color-1);
+      }
+    }
+  }
+
+  .fav-sites-entry, .favorites-term-header {
+
+    .bi-pin-fill {
+      color: var(--sakai-text-color-1);
+    }
+
+    &:hover, &:focus, button:focus {
+
+      background-color: var(--site-nav-btn-hover-bg);
+
+      .bi-pin {
+        color: var(--sakai-text-color-1);
+      }
+    }
+  }
```

Please get the indentation right, it's difficult to do it in here :)
```suggestion
  .fav-sites-entry, 
   .favorites-term-header {
      .bi-pin-fill, 
    &:hover .bi-pin, 
    &:focus .bi-pin, 
    button:focus .bi-pin {
        color: var(--sakai-text-color-1);
    }

    &:hover, 
    &:focus, 
    button:focus {
        background-color: var(--site-nav-btn-hover-bg);
    }
}
```